Application/src/visuals/Message.java

```diff
@@ -167,4 +167,16 @@ public int getToNode(){
         return toNode;
     }
 
+	@Override
+	public String getName() {
+		// Not in use.
+		return null;
```

Comment:
Shouldnt this be returning something?

Reply:
Not at the moment, had to implement it as the compiler complained about the interface lacking classes, which automatically made them unimplemented in 2 other classes.

Reply:
I think that it should be implemented if it has to be there in case someone has to use it

Reply:
interface method

Reply:
This is in the message class so it should be implemented